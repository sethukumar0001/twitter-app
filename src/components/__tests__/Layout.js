import React from 'react';
import renderer from 'react-test-renderer';
import Layout from '../Layout';

jest.mock('../../containers/Header', () => () => <h1>Header</h1>);

describe('Layout', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <Layout>
        <p>children</p>
      </Layout>,
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
