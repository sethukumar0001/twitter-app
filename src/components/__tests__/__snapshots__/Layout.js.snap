// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<main
  className="Layout-root-1"
>
  <h1>
    Header
  </h1>
  <div
    className="MuiGrid-container-3 MuiGrid-justify-xs-center-20 Layout-page-2"
  >
    <div
      className="MuiGrid-item-4 MuiGrid-grid-xs-11-42 MuiGrid-grid-sm-6-51 MuiGrid-grid-md-4-63"
    >
      <p>
        children
      </p>
    </div>
  </div>
</main>
`;
